CHANGELOG.md

```diff
@@ -1,6 +1,66 @@
+
+**Closed issues:**
+
+- Add Swift Package Manager badges to README.md [\#1204](https://github.com/ably/ably-cocoa/issues/1204)
+- NSSecureCoding NSObject bypasses security [\#1188](https://github.com/ably/ably-cocoa/issues/1188)
+- Add Ably-cocoa swift package to Swift Package Index [\#1141](https://github.com/ably/ably-cocoa/issues/1141)
+- App Store upload problem - non-public API use [\#1104](https://github.com/ably/ably-cocoa/issues/1104)
+- Memory leak\(s\) when using event emitters [\#1099](https://github.com/ably/ably-cocoa/issues/1099)
+- build warning - treating a forced downcast to 'ARTClientOptions' as optional will never produce 'nil' [\#1035](https://github.com/ably/ably-cocoa/issues/1035)
+- Warnings about deprecations when used in a Mac Catalyst app [\#1003](https://github.com/ably/ably-cocoa/issues/1003)
+- Possible race conditions in Test Suite [\#912](https://github.com/ably/ably-cocoa/issues/912)
+
+**Merged pull requests:**
+
+- Fix `ARTChannel.setOptions` and `ARTRealtimeChannel.setOptions`: Recreate ARTDataEncoder on ARTChannel options update [\#1208](https://github.com/ably/ably-cocoa/pull/1208) ([lukasz-szyszkowski](https://github.com/lukasz-szyszkowski))
+- Add Swift Package Index badges [\#1205](https://github.com/ably/ably-cocoa/pull/1205) ([lukasz-szyszkowski](https://github.com/lukasz-szyszkowski))
+- Fix/694 static analyser issues [\#1198](https://github.com/ably/ably-cocoa/pull/1198) ([maratal](https://github.com/maratal))
+- Log message text fixed. [\#1197](https://github.com/ably/ably-cocoa/pull/1197) ([maratal](https://github.com/maratal))
+- Removed optional base64 auth header [\#1195](https://github.com/ably/ably-cocoa/pull/1195) ([maratal](https://github.com/maratal))
+- Fix/1188 Added allowed class for NSKeyedUnarchiver [\#1194](https://github.com/ably/ably-cocoa/pull/1194) ([maratal](https://github.com/maratal))
+- Added Xcodebuild Logs uploading step [\#1192](https://github.com/ably/ably-cocoa/pull/1192) ([maratal](https://github.com/maratal))
+- Make sure we all use the same version of Fastlane [\#1189](https://github.com/ably/ably-cocoa/pull/1189) ([lawrence-forooghian](https://github.com/lawrence-forooghian))
+- Assure new secret to be created only together with the deviceId [\#1187](https://github.com/ably/ably-cocoa/pull/1187) ([maratal](https://github.com/maratal))
+- Feature/760 remove hardcoded error codes [\#1182](https://github.com/ably/ably-cocoa/pull/1182) ([maratal](https://github.com/maratal))
+- Replaced bundleVersion to hardcoded ARTDefault\_version [\#1181](https://github.com/ably/ably-cocoa/pull/1181) ([maratal](https://github.com/maratal))
+- Cartfile.private updated for tests execution in Xcode 13 [\#1180](https://github.com/ably/ably-cocoa/pull/1180) ([maratal](https://github.com/maratal))
+- Remove `Copyright` and `Created by` from comment header [\#1178](https://github.com/ably/ably-cocoa/pull/1178) ([lukasz-szyszkowski](https://github.com/lukasz-szyszkowski))
```

Comment:
This is an example of a change that has no customer facing benefit, so our preference is to leave these out of the change log.

Reply:
Okay, I've removed some in 0ada1c90858c63565bc72082b68c23bae50795f9